.github/workflows/test.yaml

```diff
@@ -0,0 +1,25 @@
+name: mod landing page tests
+on:
+  pull_request:
+    branches:
+      - blue_team_code
+  push:
+    branches:
+      - gh_action-mod-pages-tests
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0
+      - run: git fetch --no-tags --prune --depth=5 origin ${{ github.head_ref }}
```

Comment:
I think this line duplicates the first step, the actions/checkout@v2 ? Did you find you had to do both for this to work?

Reply:
Yep, it might do, but some docs said it might have been just getting the commit rather than the branch etc.